beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -5,29 +5,30 @@ odoo.define("beesdoo_pos.screens", function (require) {
 
     models.load_fields("res.partner", "is_company");
 
-    var set_customer_info = function (parent_class, value) {
-        var parent = this.$(parent_class);
-        var info_div = document.createElement("div");
-        info_div.textContent = value;
+    var set_customer_info = function (el_class, value, prefix) {
+        var el = this.$(el_class);
+        el.empty();
+        if (prefix && value) {
+            value = prefix + value;
+        }
         if (value) {
-            parent.append(info_div);
+            el.append(value + "<br />");
         }
```

Comment:
```suggestion
        if (prefix && value) {
            value = prefix + value;
        }
        this.$(el_class).text(value)
```

something like this? no need to empty+append when you can just set the text immediately.

Reply:
much more concise, great :+1: !

Reply:
Actually no the append is needed in case of multiple eaters, otherwise it will just show one

Reply:
OK so I moved the `empty` out of the function to trigger it only once

Reply:
@victor-champonnois I wouldn't do it like that. I would just call the function with a value of a `<br />` separated string of names. So instead of calling `set_customer_info` each time for every eater, just call it once.

Reply:
Fixed, thanks for the proposition :)